client/src/components/FrontPage.tsx

```diff
@@ -31,9 +35,16 @@ const FrontPage = (): JSX.Element => {
         <Typography variant="h1" align="left" sx={{ flexGrow: 1 }}>
           Your Current Courses
         </Typography>
+        { /* admins are shown the button for creating a new course */
+          auth.role == SystemRole.Admin &&
+          <Button id='ag_new_course_btn' size='large' variant='contained' onClick={() => {
+            navigate('/create-course');
+          }}>
+            Create New Course
+          </Button>
+        }
```

Comment:
Courses are created from `AllCoursesView` as of #254, so this should be removed.